### ISSUE TITLE
Updated people list

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#1151]
Items in the list will now include fewer items. This should be in line with #1147 .

Which items to remove TBC.

**Comment by asimonw** [comment]
@fannyBkane Rather than removing phone number and email from the list display, we could turn them into icons? The full, copy-able versions would be available on the profile.

**Comment by iteles** [comment]
Estimate TBC, awaiting further details.

**Comment by fannyBkane** [comment]
[Link to Invision - Desktop](https://projects.invisionapp.com/d/main#/console/12767821/267419073/preview)
![screen shot 2017-12-05 at 16 44 02](https://user-images.githubusercontent.com/20577339/33619039-0e71d5ba-d9dc-11e7-9871-5eb71ba592a9.png)

**Comment by asimonw** [comment]
@iteles Top comment updated and new visual added (mobile coming soon).

**Comment by fannyBkane** [comment]
[Link to Invision](https://projects.invisionapp.com/d/main#/console/12795181/268152490/comments#project_console)
![screen shot 2017-12-07 at 12 13 49](https://user-images.githubusercontent.com/20577339/33714792-25f19c80-db48-11e7-9cae-6a86b82abad0.png)

**Comment by Cleop** [comment]
@fannyBkane - where would the user type go on the mobile version for admin users?

**Comment by fannyBkane** [comment]
Hey @Cleop it could be below the job title 
![screen shot 2017-12-11 at 16 32 40](https://user-images.githubusercontent.com/20577339/33841922-f648bd0a-de90-11e7-85bd-165aed4c146b.png)

**Comment by Cleop** [comment]
Still to do:
---Cleo to look for pill icon styling and general people styling on the org profile view. Then needs to make sure this is reflected across other people lists.

**Comment by Cleop** [comment]
@fannyBkane - a couple of design inconsistencies between mobile and desktop. For now I'm going to just implement one of them as I know you may not be available today. They're only small.

Mobile:
- job title is black / desktop it's denim
- user name is bold / desktop normal

For now I'm implementing bold user name and denim font on both.

**Comment by fannyBkane** [comment]
Sounds good @Cleop — just one tiny thing, the denim is the functional colour for links,  and as such items that aren't links will not be in that colour on the designs.

**Comment by Cleop** [comment]
@fannyBkane - in these two shots from invision there are words in denim that aren't links:

<img src="https://user-images.githubusercontent.com/16775804/34055772-9ed5474c-e1c8-11e7-85e2-daf2f02c5026.png" width=200px />
<img src="https://user-images.githubusercontent.com/16775804/34055811-c395e6ae-e1c8-11e7-9026-aa30e4667782.png" width=700px />

'at' in the first one is not a link and 'CEO at' is not in the second one. From what you've said above would you like me to ignore this and make them black? or would you like the 'at' to be part of the link?

I thought the reason why in another issue we discussed making user's names italics was because otherwise this bit would look like one big link:
![image](https://user-images.githubusercontent.com/16775804/34055911-1e352548-e1c9-11e7-807f-87973e448a1c.png)

Would you like the 'at' here to be black too? If so, would you still like the name to be in italics or would the black word in-between the two links be enough to separate them?

**Comment by fannyBkane** [comment]
@Cleop that's right. In that case, let's make 'at' the #333 grey in all instances, but no need to change the italic on mobile.

**Comment by iteles** [comment]
@Cleop We noted that the denim links are updated in the People list under the Org but needs to be updated in the Explore page.

**Comment by asimonw** [comment]
@Cleop Shall I wait to test this until you've changed those links to denim?
I'm also assuming that the email icon will also get increased in size, as in #1149 ?

**Comment by asimonw** [comment]
Everything else looks 💯

**Comment by Cleop** [comment]
Both of these things are in #1204 - so should be ready once that is merged.

**Comment by fannyBkane** [comment]
@Cleop having tapped either the call or the email button, the whole region was highlighted (and stayed highlighted after cancelling the action). Could the whole region highlight be removed? be removed? ![img_0790](https://user-images.githubusercontent.com/20577339/34518865-595a7e8e-f079-11e7-8b36-9cbd52230a5d.PNG)

**Comment by asimonw** [comment]
@fannyBkane That's the desktop hover effect. On mobile, tap events trigger this hover effect by default. We could only remove the effect for smaller screen sizes so that the hover effect still works on desktop. wdyt?

**Comment by fannyBkane** [comment]
@asimonw thanks for clarifying that's actually what I meant :joy:
